files/zh-cn/web/javascript/reference/global_objects/set/clear/index.md

```diff
@@ -32,7 +38,7 @@ console.log(mySet.has("foo")); // true
 mySet.clear();
 
 console.log(mySet.size); // 0
-console.log(mySet.has("foo")); // false
+console.log(mySet.has("bar")); // false
```

Comment:
I think it's better to use "foo" (not "bar")

Reply:
No problem.

Reply:
In https://github.com/mdn/content/pull/29586.